test/mocks.h

```diff
@@ -33,6 +33,8 @@ typedef struct mock_os_data_obj
    uint16_t eth_send_len;
    uint16_t eth_send_count;
 
+   pnal_eth_status_t eth_status[PNET_MAX_PORT + 1];
```

Comment:
Maybe add a comment that `local_port_num` starts at 1

Reply:
Done.